developer/src/kmc/src/keyman/compiler/meta.ts

```diff
@@ -24,9 +24,13 @@ export class MetaCompiler extends SectionCompiler {
     if(versionNumber !== undefined) {
```

Comment:
Should we also report `CompilerMessages.Error_InvalidVersion()` if versionNumber is undefined?

Reply:
No, it's an optional field. If not present, defaults to `0.0.0`.